scripts/vendor

```diff
@@ -60,8 +68,21 @@ case $TYP in
     init
     outdated
     ;;
+  "clean")
+    clean
+    exit 0
+    ;;
   *)
-    echo >&2 "Unknown type $TYP"
-    exit 1
+    usage
     ;;
 esac
+
+while getopts ":c" opt; do
+  case $opt in
+    c)
+      clean
+      ;;
+    *)
+      ;;
+  esac
+done
```

Comment:
Perhaps we should reverse the option; by default, remove the `go.mod`, and only if it's needed after it ran, pass an option to `--keep` (or whatever) the file.

Looking at the changes, I think all (but one) calls of this script expect the `go.mod` to be deleted afterwards; WDYT?